src/dlgTriggerEditor.cpp

```diff
@@ -4757,42 +4763,63 @@ void dlgTriggerEditor::slot_set_pattern_type_color(int type)
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 1:
         palette.setColor(QPalette::Text, QColor(Qt::blue));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 2:
         palette.setColor(QPalette::Text, QColor(195, 0, 0));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 3:
         palette.setColor(QPalette::Text, QColor(0, 195, 0));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 4:
         palette.setColor(QPalette::Text, QColor(0, 155, 155));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 5:
         palette.setColor(QPalette::Text, QColor(137, 0, 205));
         pItem->lineEdit_pattern->show();
         pItem->pushButton_fgColor->hide();
         pItem->pushButton_bgColor->hide();
+        pItem->pushButton_prompt->hide();
         break;
     case 6:
         palette.setColor(QPalette::Text, QColor(100, 100, 100));
         pItem->lineEdit_pattern->hide();
         pItem->pushButton_fgColor->show();
         pItem->pushButton_bgColor->show();
+        pItem->pushButton_prompt->hide();
+        break;
+    case 7:
+        palette.setColor(QPalette::Text, QColor(Qt::black));
+        pItem->lineEdit_pattern->hide();
+        pItem->pushButton_fgColor->hide();
+        pItem->pushButton_bgColor->hide();
+        if (mpHost->mTelnet.mGA_Driver) {
```

Comment:
Is `mpHost->mTelnet.mGA_Driver` the right variable to work with, or should it be `mpHost->mTelnet.recvdGA`?

Reply:
Yeah that is the right variable, it tracks the status of whenever we are working with a GA game in general while the latter is whenever we've received the signal for that particular batch of data.

Reply:
Looking at the code in `cTelnet` it seems that `cTelent::GA_Driver` does go up and down during the session - will it always be in the `true` state when the code in the `dlgTriggerEditor` class checks it with a Server using GA?

Reply:
I've tested it and it seems to work how I'd like it to, and `recvdGA` is certainly not the variable I need, so I'm not sure what are you trying to get at here 🤔

Reply:
As I am not familiar with a Server that uses it (isn't it an IRE thing?) I was unsure of precisely the variable was set true - all I could see was that it seemed to be set and reset during the normal flow of data through the `cTelnet` class and I wasn't sure that it would always be `true` when the editor part considered it - if you have tested it and found that it works then I guess it is OK... :blush:

Reply:
Of course, I always test my work!

Reply:
It's not IRE-specific but IRE does make use of it, and that enables Mudlets `isPrompt()` to work. Ideally more games would support it too.